### ISSUE TITLE
Error when trying to convert

### ISSUE THREAD

**Issue by ChronoStriker1** [DrWhoCares/imgdanke#86]
it looks like you may have hardcoded a user (DrWhoCares) path as Im getting an error when the program is creating a temp file

```See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.IO.FileNotFoundException: Could not find file 'C:\Users\chronostriker1\Desktop/__danketmp/YS_CH033_001.png'.
File name: 'C:\Users\chronostriker1\Desktop/__danketmp/YS_CH033_001.png'
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileInfo.get_Length()
   at imgdanke.MainForm.UpdateFileInfosToTempFolder(List`1& imgFiles, String tempFolderPath, Int64& previousTotalFilesize) in C:\Users\DrWhoCares\Desktop\Translation Utilities\imgdanke\imgdanke\MainForm.cs:line 2062
   at imgdanke.MainForm.ProcessFiles(List`1& imgFiles, String tempFolderPath) in C:\Users\DrWhoCares\Desktop\Translation Utilities\imgdanke\imgdanke\MainForm.cs:line 1941
   at imgdanke.MainForm.StartButton_Click(Object sender, EventArgs e) in C:\Users\DrWhoCares\Desktop\Translation Utilities\imgdanke\imgdanke\MainForm.cs:line 1706
   at System.Windows.Forms.Control.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.Button.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)


************** Loaded Assemblies **************
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4300.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework64/v4.0.30319/mscorlib.dll
----------------------------------------
imgdanke
    Assembly Version: 1.2021.3.1
    Win32 Version: 1.2021.3.1
    CodeBase: file:///C:/imgdanke/imgdanke.exe
----------------------------------------
System.Windows.Forms
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4270.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4300.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Drawing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3752.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System.Text.Json
    Assembly Version: 5.0.0.1
    Win32 Version: 5.0.220.61120
    CodeBase: file:///C:/imgdanke/lib/System.Text.Json.DLL
----------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4190.0 built by: NET48REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4320.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
System.Xml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3752.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
System.Memory
    Assembly Version: 4.0.1.1
    Win32 Version: 4.6.28619.01
    CodeBase: file:///C:/imgdanke/lib/System.Memory.DLL
----------------------------------------
Microsoft.Bcl.AsyncInterfaces
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/imgdanke/lib/Microsoft.Bcl.AsyncInterfaces.DLL
----------------------------------------
System.Threading.Tasks.Extensions
    Assembly Version: 4.2.0.1
    Win32 Version: 4.6.28619.01
    CodeBase: file:///C:/imgdanke/lib/System.Threading.Tasks.Extensions.DLL
----------------------------------------
System.Text.Encodings.Web
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/imgdanke/lib/System.Text.Encodings.Web.DLL
----------------------------------------
System.ValueTuple
    Assembly Version: 4.0.3.0
    Win32 Version: 4.6.26515.06
    CodeBase: file:///C:/imgdanke/lib/System.ValueTuple.DLL
----------------------------------------
System.Runtime.CompilerServices.Unsafe
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.20.51904
    CodeBase: file:///C:/imgdanke/lib/System.Runtime.CompilerServices.Unsafe.DLL
----------------------------------------
System.Buffers
    Assembly Version: 4.0.3.0
    Win32 Version: 4.6.28619.01
    CodeBase: file:///C:/imgdanke/lib/System.Buffers.DLL
----------------------------------------
Onova
    Assembly Version: 2.6.2.0
    Win32 Version: 2.6.2.0
    CodeBase: file:///C:/imgdanke/lib/Onova.DLL
----------------------------------------
System.Net.Http
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3752.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Net.Http/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Net.Http.dll
----------------------------------------
System.Runtime.InteropServices.RuntimeInformation
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3752.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Runtime.InteropServices.RuntimeInformation/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Runtime.InteropServices.RuntimeInformation.dll
----------------------------------------
System.Numerics.Vectors
    Assembly Version: 4.1.4.0
    Win32 Version: 4.6.26515.06
    CodeBase: file:///C:/imgdanke/lib/System.Numerics.Vectors.DLL
----------------------------------------
Microsoft.WindowsAPICodePack.Shell
    Assembly Version: 1.1.0.0
    Win32 Version: 1.1.0.0
    CodeBase: file:///C:/imgdanke/lib/Microsoft.WindowsAPICodePack.Shell.DLL
----------------------------------------
Microsoft.WindowsAPICodePack
    Assembly Version: 1.1.2.0
    Win32 Version: 1.1.2.0
    CodeBase: file:///C:/imgdanke/lib/Microsoft.WindowsAPICodePack.DLL
----------------------------------------
PresentationFramework
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4320.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/PresentationFramework/v4.0_4.0.0.0__31bf3856ad364e35/PresentationFramework.dll
----------------------------------------
WindowsBase
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4320.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/WindowsBase/v4.0_4.0.0.0__31bf3856ad364e35/WindowsBase.dll
----------------------------------------
PresentationCore
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4320.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_64/PresentationCore/v4.0_4.0.0.0__31bf3856ad364e35/PresentationCore.dll
----------------------------------------
System.Xaml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4320.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xaml/v4.0_4.0.0.0__b77a5c561934e089/System.Xaml.dll
----------------------------------------
Accessibility
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3752.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/Accessibility/v4.0_4.0.0.0__b03f5f7f11d50a3a/Accessibility.dll
----------------------------------------

************** JIT Debugging **************
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.

For example:

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.```

**Comment by DrWhoCares** [comment]
Could you please edit your post to put the error in a code block? Use three backticks at the beginning and end, like this ```.

I believe the hardcoded path you're seeing is actually just the build information that's packaged into the .pdb.
I'm not sure exactly what the issue is, and can't do much debugging until after work.
Please let me know the settings you were using, or upload the UserConfig file so I can view it.

**Comment by DrWhoCares** [comment]
@ChronoStriker1 Question, are you working across drives? As in, for example, is your Source Folder Path set to a location on your D:\ drive and your Output Folder Path is a location on your C:\ drive?

**Comment by ChronoStriker1** [comment]
In this instance yes

**Comment by DrWhoCares** [comment]
In that case, that's the reason why.
It'll be fixed later today.

**Comment by DrWhoCares** [comment]
Should be fixed in 1600b6ea
Released in v1.2021.3.4
Should be the same as #87
